examples/settings.py

```diff
@@ -153,10 +152,8 @@ def create_fisheye_spec(**kw_args):
 
         fisheye_sensor_spec.resolution = [settings["height"], settings["width"]]
         # The default principal_point_offset is the middle of the image
-        fisheye_sensor_spec.principal_point_offset = mn.Vector2(
-            settings["height"] // 2,
-            settings["width"] // 2,
-        )
+        fisheye_sensor_spec.principal_point_offset = None
+        # default: fisheye_sensor_spec.principal_point_offset = [i/2 for i in fisheye_sensor_spec.resolution]
```

Comment:
typo: 1/2 not i/2

Reply:
No, this intended: 
```
[i/2 for i in fisheye_sensor_spec.resolution]
``` 
is a valid list comprehension.